.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "isl"]
 	path = isl
 	url = https://github.com/inducer/isl.git
+[submodule "wheels/multibuild"]
+	path = wheels/multibuild
+	url = https://github.com/matthew-brett/multibuild
```

Comment:
Is this submodule avoidable? (Maybe by cloning during the build?) I ask because it is essentially dead weight for normal dev checkouts.